src/test/scala/org/broadinstitute/dsde/workbench/sam/service/TosServiceSpec.scala

```diff
@@ -73,4 +67,38 @@ class TosServiceSpec extends AnyFlatSpec with TestSupport with BeforeAndAfterAll
     tosServiceEnabledV2.acceptTosStatus(defaultUser.id, samRequestContext).unsafeRunSync() shouldBe Option(true)
     tosServiceEnabledV2.getTosStatus(defaultUser.id, samRequestContext).unsafeRunSync() shouldBe Option(true)
   }
+
+  "TosService.isTermsOfServiceStatusAcceptable" should "allow all requests to the API if TOS is disabled" in {
+    assume(databaseEnabled, databaseEnabledClue)
+    dirDAO.createUser(defaultUser, samRequestContext).unsafeRunSync()
```

Comment:
nit: maybe instead of defaultUser it could be named more specifically to its tos state?

Reply:
`defaultUser` is used because that's the one generated for each test case on line 16

Reply:
I have a slight preference towards generating a new test user within the test too, since that is what we are testing here, but I would not hold up the PR for this fix, especially since the way you are doing it here is the pattern used by all the other existing tests.  However, going forward, I do think it makes the test a little more resilient to refactoring and less likely to conflict with other tests if we (within reason) can keep the tests well isolated and sharing (subjectively) appropriate setup/fixture logic.  This is one of the skills we need to sharpen, when is or is not the right time to do this.